soc/arm/nxp_imx/rt/Kconfig.soc

```diff
@@ -36,6 +36,7 @@ config SOC_MIMXRT1021
 	select HAS_MCUX_LPI2C
 	select HAS_MCUX_LPSPI
 	select HAS_MCUX_LPUART
+	select HAS_MCUX_GPT
```

Comment:
Can you add this to `SOC_MIMXRT1015` too please?

Reply:
done